dropwizard/service/src/test/java/org/apache/polaris/service/dropwizard/catalog/TestUtil.java

```diff
@@ -179,4 +179,22 @@ public static RESTCatalog createSnowmanManagedCatalog(
     restCatalog.initialize("polaris", propertiesBuilder.buildKeepingLast());
     return restCatalog;
   }
+
+  /**
+   * Asserts that the response has the expected status code, with a custom fail message. The
+   * response entity is buffered so it can be read multiple times.
+   *
+   * @param response The response to check
+   * @param expectedStatusCode The expected status code
+   */
+  private static void assertStatusCodeWithFailMessage(Response response, int expectedStatusCode) {
```

Comment:
I'd rename this to just `assertStatusCode` - the current name made me think it was checking the response for an expected failure message and I was very confused about why these calls were suddenly failing.

Reply:
+1, actually I also did a double take initially when reading this, but was just on the fence about whether it was just my own preconceived notions :)

Reply:
This makes sense! Renamed the helper method name to make it less confusing :)